src/app/components/auth-callback/auth-callback.component.css.map

```diff
@@ -0,0 +1,9 @@
+{
+    "version": 3,
+    "mappings": "AAAA;;;;;;;;;;;;;;;;;GAiBG",
+    "sources": [
+        "auth-callback.component.scss"
+    ],
+    "names": [],
+    "file": "auth-callback.component.css"
+}
```

Comment:
What does this file do? I guess it maps the .css to .scss? Why not use .scss?

Reply:
I think it came from a SCSS plugin in my IDE. I removed it